### ISSUE TITLE
Fatal crash with 11.4

### ISSUE THREAD

**Issue by ErichHeer** [syncthing/syncthing#1814]
After updating to 11.4 from 11.3 I get a fatal crash with syncthing. 
Syncthing crashes not Windows. (Nothing else involved in the crash - just syncthing).

**Comment by pw1** [comment]
I have the same problem after upgrading on Xubuntu 15.04.

Here is the crash log:
$ /home/paul/opt/syncthing-linux-amd64-v0.11.4/syncthing -no-browser
[W5TRX] 11:53:22 INFO: syncthing v0.11.4 (go1.4.2 linux-amd64 default) unknown-user@syncthing-builder 2015-05-15 08:46:47 UTC
[W5TRX] 11:53:22 INFO: My ID: W5TRXIL-7YD3B6P-THV7E5X-RTQYXUD-E23SHJ6-SHNZ6LJ-PHP2UEK-IWY2IQI
[W5TRX] 11:53:22 INFO: Database block cache capacity 47745 KiB
[W5TRX] 11:53:23 INFO: Starting web GUI on http://127.0.0.1:8384/
[W5TRX] 11:53:23 OK: Ready to synchronize sync (read-write)
[W5TRX] 11:53:23 OK: Ready to synchronize todo (read-write)
[W5TRX] 11:53:23 INFO: Device A7C4AV4-RRZ32SW-R4PYIPI-Z3S3IWU-ZOTT6UP-XKRMRQA-DPQBHVO-J4TQYQH is "Defiance Bay" at [defbay.wagenaars.org:38902]
[W5TRX] 11:53:23 INFO: Device W5TRXIL-7YD3B6P-THV7E5X-RTQYXUD-E23SHJ6-SHNZ6LJ-PHP2UEK-IWY2IQI is "Tar Valon" at [dynamic]
[W5TRX] 11:53:23 INFO: Starting usage reporting
[W5TRX] 11:53:23 INFO: Completed initial scan (rw) of folder todo
[W5TRX] 11:53:23 INFO: Completed initial scan (rw) of folder sync
SIGILL: illegal instruction
PC=0x749e3e

goroutine 23 [running]:
crypto/elliptic.p256MovCond(0xc20805ed20, 0xc, 0xc, 0xc20805ed20, 0xc, 0xc, 0xc20805ec60, 0xc, 0xc, 0xe, ...)
	/usr/local/go/src/crypto/elliptic/p256_asm_amd64.s:82 +0x1e fp=0xc20843cc98 sp=0xc20843cc90
crypto/elliptic.p256ScalarMult(0xc20805ec60, 0xc20805ec60, 0xc20843cdb0, 0x4, 0x4)
	/usr/local/go/src/crypto/elliptic/p256_amd64.go:461 +0x10c2 fp=0xc20843cd68 sp=0xc20843cc98
crypto/elliptic.p256Curve.ScalarMult(0xc2082b0b10, 0xc208442460, 0xc208442480, 0xc208442d80, 0x20, 0x20, 0x20, 0xc208442e40)
	/usr/local/go/src/crypto/elliptic/p256_amd64.go:228 +0x36f fp=0xc20843ce28 sp=0xc20843cd68
crypto/tls.(*ecdheKeyAgreement).generateClientKeyExchange(0xc208113400, 0xc208064ea0, 0xc208234280, 0xc208438480, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/usr/local/go/src/crypto/tls/key_agreement.go:400 +0x306 fp=0xc20843cfa0 sp=0xc20843ce28
crypto/tls.(*clientHandshakeState).doFullHandshake(0xc2084302d0, 0x0, 0x0)
	/usr/local/go/src/crypto/tls/handshake_client.go:415 +0x12fd fp=0xc20843d538 sp=0xc20843cfa0
crypto/tls.(*Conn).clientHandshake(0xc20810a2c0, 0x0, 0x0)
	/usr/local/go/src/crypto/tls/handshake_client.go:197 +0x1981 fp=0xc20843d9b8 sp=0xc20843d538
crypto/tls.(*Conn).Handshake(0xc20810a2c0, 0x0, 0x0)
	/usr/local/go/src/crypto/tls/conn.go:977 +0xf1 fp=0xc20843d9f8 sp=0xc20843d9b8
main.(*connectionSvc).connect(0xc20802e5f0)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/connections.go:304 +0xb68 fp=0xc20843df28 sp=0xc20843d9f8
main.*connectionSvc.(main.connect)·fm()
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/connections.go:80 +0x27 fp=0xc20843df38 sp=0xc20843df28
main.serviceFunc.Serve(0xc20802af20)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/summarysvc.go:202 +0x20 fp=0xc20843df40 sp=0xc20843df38
github.com/thejerf/suture.func·007()
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/thejerf/suture/suture.go:516 +0xf6 fp=0xc20843dfe0 sp=0xc20843df40
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:2232 +0x1 fp=0xc20843dfe8 sp=0xc20843dfe0
created by github.com/thejerf/suture.(*Supervisor).runService
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/thejerf/suture/suture.go:519 +0x11e

goroutine 1 [chan receive]:
main.syncthingMain()
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/main.go:706 +0x3175
main.main()
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/main.go:378 +0x21cc

goroutine 5 [syscall]:
os/signal.loop()
	/usr/local/go/src/os/signal/signal_unix.go:21 +0x1f
created by os/signal.init·1
	/usr/local/go/src/os/signal/signal_unix.go:27 +0x35

goroutine 7 [chan receive]:
main.trackCPUUsage()
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/gui_unix.go:24 +0xec
created by main.init·2
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/gui_unix.go:17 +0x25

goroutine 8 [select]:
github.com/thejerf/suture.(*Supervisor).Serve(0xc2080ec000)
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/thejerf/suture/suture.go:411 +0xf64
created by github.com/thejerf/suture.(*Supervisor).ServeBackground
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/thejerf/suture/suture.go:373 +0x32

goroutine 10 [chan receive]:
github.com/syncthing/syncthing/internal/config.(*Wrapper).Serve(0xc2080de200)
	/go/src/github.com/syncthing/syncthing/internal/config/wrapper.go:84 +0x75
created by github.com/syncthing/syncthing/internal/config.Wrap
	/go/src/github.com/syncthing/syncthing/internal/config/wrapper.go:59 +0x28d

goroutine 11 [select]:
github.com/syndtr/goleveldb/leveldb/util.(*BufferPool).drain(0xc2080ae0e0)
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/syndtr/goleveldb/leveldb/util/buffer_pool.go:205 +0x21e
created by github.com/syndtr/goleveldb/leveldb/util.NewBufferPool
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/syndtr/goleveldb/leveldb/util/buffer_pool.go:236 +0x24c

goroutine 12 [select]:
github.com/syndtr/goleveldb/leveldb.(*DB).compactionError(0xc2081262c0)
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/syndtr/goleveldb/leveldb/db_compaction.go:120 +0x48c
created by github.com/syndtr/goleveldb/leveldb.openDB
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/syndtr/goleveldb/leveldb/db.go:126 +0x849

goroutine 13 [select]:
github.com/syndtr/goleveldb/leveldb.(*DB).mpoolDrain(0xc2081262c0)
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/syndtr/goleveldb/leveldb/db_state.go:73 +0x14a
created by github.com/syndtr/goleveldb/leveldb.openDB
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/syndtr/goleveldb/leveldb/db.go:127 +0x863

goroutine 14 [select]:
github.com/syndtr/goleveldb/leveldb.(*DB).tCompaction(0xc2081262c0)
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/syndtr/goleveldb/leveldb/db_compaction.go:812 +0x858
created by github.com/syndtr/goleveldb/leveldb.openDB
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/syndtr/goleveldb/leveldb/db.go:130 +0x8a7

goroutine 15 [select]:
github.com/syndtr/goleveldb/leveldb.(*DB).mCompaction(0xc2081262c0)
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/syndtr/goleveldb/leveldb/db_compaction.go:759 +0x283
created by github.com/syndtr/goleveldb/leveldb.openDB
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/syndtr/goleveldb/leveldb/db.go:131 +0x8c1

goroutine 16 [select]:
github.com/syndtr/goleveldb/leveldb.(*DB).jWriter(0xc2081262c0)
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/syndtr/goleveldb/leveldb/db_write.go:37 +0x197
created by github.com/syndtr/goleveldb/leveldb.openDB
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/syndtr/goleveldb/leveldb/db.go:132 +0x8db

goroutine 17 [select]:
github.com/syncthing/syncthing/internal/model.(*ProgressEmitter).Serve(0xc20803b280)
	/go/src/github.com/syncthing/syncthing/internal/model/progressemitter.go:49 +0x8f2
created by github.com/syncthing/syncthing/internal/model.NewModel
	/go/src/github.com/syncthing/syncthing/internal/model/model.go:125 +0x93c

goroutine 18 [IO wait]:
net.(*pollDesc).Wait(0xc2099219c0, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:84 +0x47
net.(*pollDesc).WaitRead(0xc2099219c0, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:89 +0x43
net.(*netFD).accept(0xc209921960, 0x0, 0x7f277b69fe00, 0xc20810e360)
	/usr/local/go/src/net/fd_unix.go:419 +0x40b
net.(*TCPListener).AcceptTCP(0xc20802c320, 0xc20823c080, 0x0, 0x0)
	/usr/local/go/src/net/tcpsock_posix.go:234 +0x4e
net.(*TCPListener).Accept(0xc20802c320, 0x0, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/tcpsock_posix.go:244 +0x4c
main.(*DowngradingListener).Accept(0xc209929a60, 0x0, 0x0, 0x0, 0x0)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/tls.go:99 +0x85
net/http.(*Server).Serve(0xc208184000, 0x7f277b6a89d8, 0xc209929a60, 0x0, 0x0)
	/usr/local/go/src/net/http/server.go:1728 +0x92
main.(*apiSvc).Serve(0xc2080b0300)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/gui.go:215 +0x190a
github.com/thejerf/suture.func·007()
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/thejerf/suture/suture.go:516 +0xf6
created by github.com/thejerf/suture.(*Supervisor).runService
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/thejerf/suture/suture.go:519 +0x11e

goroutine 19 [select]:
github.com/syncthing/syncthing/internal/events.(*Subscription).Poll(0xc209929ac0, 0xdf8475800, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/syncthing/syncthing/internal/events/events.go:178 +0x356
github.com/syncthing/syncthing/internal/events.(*BufferedSubscription).pollingLoop(0xc209927950)
	/go/src/github.com/syncthing/syncthing/internal/events/events.go:215 +0x48
created by github.com/syncthing/syncthing/internal/events.NewBufferedSubscription
	/go/src/github.com/syncthing/syncthing/internal/events/events.go:209 +0x292

goroutine 20 [select]:
github.com/syncthing/syncthing/internal/ignore.(*Matcher).clean(0xc20802f680, 0x68c61714000)
	/go/src/github.com/syncthing/syncthing/internal/ignore/ignore.go:161 +0x1c9
created by github.com/syncthing/syncthing/internal/ignore.New
	/go/src/github.com/syncthing/syncthing/internal/ignore/ignore.go:53 +0x170

goroutine 21 [select]:
github.com/syncthing/syncthing/internal/ignore.(*Matcher).clean(0xc20802e550, 0x68c61714000)
	/go/src/github.com/syncthing/syncthing/internal/ignore/ignore.go:161 +0x1c9
created by github.com/syncthing/syncthing/internal/ignore.New
	/go/src/github.com/syncthing/syncthing/internal/ignore/ignore.go:53 +0x170

goroutine 22 [select]:
github.com/thejerf/suture.(*Supervisor).Serve(0xc2080ec0c0)
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/thejerf/suture/suture.go:411 +0xf64
github.com/thejerf/suture.func·007()
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/thejerf/suture/suture.go:516 +0xf6
created by github.com/thejerf/suture.(*Supervisor).runService
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/thejerf/suture/suture.go:519 +0x11e

goroutine 24 [IO wait]:
net.(*pollDesc).Wait(0xc208224370, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:84 +0x47
net.(*pollDesc).WaitRead(0xc208224370, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:89 +0x43
net.(*netFD).accept(0xc208224310, 0x0, 0x7f277b69fe00, 0xc20816a6e0)
	/usr/local/go/src/net/fd_unix.go:419 +0x40b
net.(*TCPListener).AcceptTCP(0xc208274160, 0x7f277b6a77d8, 0x0, 0x0)
	/usr/local/go/src/net/tcpsock_posix.go:234 +0x4e
net.(*TCPListener).Accept(0xc208274160, 0x0, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/tcpsock_posix.go:244 +0x4c
main.(*connectionSvc).listen(0xc20802e5f0, 0xc20810f450, 0xd)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/connections.go:219 +0x4d6
main.func·001()
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/connections.go:84 +0x37
main.serviceFunc.Serve(0xc208168500)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/summarysvc.go:202 +0x20
github.com/thejerf/suture.func·007()
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/thejerf/suture/suture.go:516 +0xf6
created by github.com/thejerf/suture.(*Supervisor).runService
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/thejerf/suture/suture.go:519 +0x11e

goroutine 25 [chan receive]:
main.(*connectionSvc).handle(0xc20802e5f0)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/connections.go:95 +0x7e
main.*connectionSvc.(main.handle)·fm()
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/connections.go:88 +0x27
main.serviceFunc.Serve(0xc20802af50)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/summarysvc.go:202 +0x20
github.com/thejerf/suture.func·007()
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/thejerf/suture/suture.go:516 +0xf6
created by github.com/thejerf/suture.(*Supervisor).runService
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/thejerf/suture/suture.go:519 +0x11e

goroutine 33 [syscall, locked to thread]:
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:2232 +0x1

goroutine 34 [chan receive]:
github.com/syncthing/syncthing/internal/versioner.func·001()
	/go/src/github.com/syncthing/syncthing/internal/versioner/staggered.go:80 +0xba
created by github.com/syncthing/syncthing/internal/versioner.NewStaggered
	/go/src/github.com/syncthing/syncthing/internal/versioner/staggered.go:83 +0x6dd

goroutine 35 [select]:
github.com/syncthing/syncthing/internal/model.(*rwFolder).Serve(0xc20806c300)
	/go/src/github.com/syncthing/syncthing/internal/model/rwfolder.go:150 +0x1e27
created by github.com/syncthing/syncthing/internal/model.(*Model).StartFolderRW
	/go/src/github.com/syncthing/syncthing/internal/model/model.go:166 +0x570

goroutine 36 [chan receive]:
github.com/syncthing/syncthing/internal/versioner.func·001()
	/go/src/github.com/syncthing/syncthing/internal/versioner/staggered.go:80 +0xba
created by github.com/syncthing/syncthing/internal/versioner.NewStaggered
	/go/src/github.com/syncthing/syncthing/internal/versioner/staggered.go:83 +0x6dd

goroutine 37 [select]:
github.com/syncthing/syncthing/internal/model.(*rwFolder).Serve(0xc20806c700)
	/go/src/github.com/syncthing/syncthing/internal/model/rwfolder.go:150 +0x1e27
created by github.com/syncthing/syncthing/internal/model.(*Model).StartFolderRW
	/go/src/github.com/syncthing/syncthing/internal/model/model.go:166 +0x570

goroutine 38 [select]:
github.com/thejerf/suture.(*Supervisor).Serve(0xc20811a000)
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/thejerf/suture/suture.go:411 +0xf64
created by github.com/thejerf/suture.(*Supervisor).ServeBackground
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/thejerf/suture/suture.go:373 +0x32

goroutine 43 [select]:
main.(*usageReportingService).Serve(0xc208272400)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/usage_report.go:149 +0x48f
github.com/thejerf/suture.func·007()
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/thejerf/suture/suture.go:516 +0xf6
created by github.com/thejerf/suture.(*Supervisor).runService
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/thejerf/suture/suture.go:519 +0x11e

goroutine 54 [sleep]:
main.standbyMonitor()
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/main.go:965 +0x64
created by main.syncthingMain
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/main.go:688 +0x30a1

goroutine 55 [sleep]:
main.generatePingEvents()
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/main.go:846 +0x2d
created by main.syncthingMain
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/main.go:702 +0x3139

goroutine 66 [select]:
github.com/thejerf/suture.(*Supervisor).Serve(0xc2080ec300)
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/thejerf/suture/suture.go:411 +0xf64
created by github.com/thejerf/suture.(*Supervisor).ServeBackground
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/thejerf/suture/suture.go:373 +0x32

goroutine 67 [select]:
main.(*folderSummarySvc).listenForUpdates(0xc20810cea0)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/summarysvc.go:67 +0x5c6
main.*folderSummarySvc.(main.listenForUpdates)·fm()
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/summarysvc.go:47 +0x27
main.serviceFunc.Serve(0xc20802b320)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/summarysvc.go:202 +0x20
github.com/thejerf/suture.func·007()
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/thejerf/suture/suture.go:516 +0xf6
created by github.com/thejerf/suture.(*Supervisor).runService
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/thejerf/suture/suture.go:519 +0x11e

goroutine 68 [select]:
main.(*folderSummarySvc).calculateSummaries(0xc20810cea0)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/summarysvc.go:115 +0x2e4
main.*folderSummarySvc.(main.calculateSummaries)·fm()
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/summarysvc.go:48 +0x27
main.serviceFunc.Serve(0xc20802b330)
	/go/src/github.com/syncthing/syncthing/cmd/syncthing/summarysvc.go:202 +0x20
github.com/thejerf/suture.func·007()
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/thejerf/suture/suture.go:516 +0xf6
created by github.com/thejerf/suture.(*Supervisor).runService
	/go/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/thejerf/suture/suture.go:519 +0x11e

rax     0xc20805ed20
rbx     0xe
rcx     0xc20805ec60
rdx     0xc
rdi     0xc20805ed20
rsi     0xc20805ed20
rbp     0xc20805ec60
rsp     0xc20843cc90
r8      0xdb628eaeaea8d419
r9      0x27a27f0afd57c728
r10     0x70329829ca6ac4bb
r11     0x676d67a676531ce9
r12     0xdb628eaeaea8d419
r13     0x27a27f0afd57c728
r14     0xe
r15     0x89f07d04dee0ad22
rip     0x749e3e
rflags  0x10202
cs      0x33
fs      0x0
gs      0x0

**Comment by AudriusButkevicius** [comment]
@ErichHeer post crash log.
@pw1 cpu model?

**Comment by pw1** [comment]
$ lscpu
Architecture:          x86_64
CPU op-mode(s):        32-bit, 64-bit
Byte Order:            Little Endian
CPU(s):                4
On-line CPU(s) list:   0-3
Thread(s) per core:    1
Core(s) per socket:    4
Socket(s):             1
NUMA node(s):          1
Vendor ID:             AuthenticAMD
CPU family:            16
Model:                 4
Model name:            AMD Phenom(tm) II X4 810 Processor
Stepping:              2
CPU MHz:               1400.000
CPU max MHz:           2600.0000
CPU min MHz:           800.0000
BogoMIPS:              5200.54
Virtualization:        AMD-V
L1d cache:             64K
L1i cache:             64K
L2 cache:              512K
L3 cache:              4096K
NUMA node0 CPU(s):     0-3

**Comment by JaCoB1123** [comment]
I get the same error with 11.4
OS: Ubuntu Server 14.04.2 LTS
CPU:  AMD Phenom(tm) II X6 1090T Processor

    > uname -a
    Linux nas 3.13.0-51-generic #84-Ubuntu SMP Wed Apr 15 12:08:34 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux

Log: https://api.javil.eu/syncthing.log

**Comment by j0j0j0** [comment]
seems that I have the same error as pw1:
SIGILL: illegal instruction
PC=0x749e3e

lscpu: (Xen VM)
Architecture:          x86_64
CPU op-mode(s):        32-bit, 64-bit
Byte Order:            Little Endian
CPU(s):                1
On-line CPU(s) list:   0
Thread(s) per core:    1
Core(s) per socket:    1
Socket(s):             1
Vendor ID:             AuthenticAMD
CPU family:            16
Model:                 5
Model name:            AMD Athlon(tm) II X3 450 Processor
Stepping:              3
CPU MHz:               3200.252
BogoMIPS:              6400.50
Hypervisor vendor:     Xen
Virtualization type:   para
L1d cache:             64K
L1i cache:             64K
L2 cache:              512K

**Comment by fablue** [comment]
Same Problem here:
Architecture:          x86_64
CPU op-mode(s):        32-bit, 64-bit
Byte Order:            Little Endian
CPU(s):                2
On-line CPU(s) list:   0,1
Thread(s) pro Kern:    1
Kern(e) pro Socket:    2
Socket(s):             1
NUMA-Knoten:           1
Anbieterkennung:       AuthenticAMD
Prozessorfamilie:      16
Modell:                6
Stepping:              3
CPU MHz:               1300.000
BogoMIPS:              4392.65
Virtualisierung:       AMD-V
L1d Cache:             64K
L1i Cache:             64K
L2 Cache:              1024K
NUMA node0 CPU(s):     0,1

**Comment by retep998** [comment]
Getting a fatal crash here on a Windows 7 x64 machine with an AMD Phenom x6 1100T
Log: https://gist.github.com/21def06b43b490bb5350
Meanwhile a Windows 8.1 x64 machine with an AMD A6-6600K is having no issues.

**Comment by LazyGatto** [comment]
Same error after autoupdate to 11.4 from 11.3 :(
Architecture:          x86_64
CPU op-mode(s):        32-bit, 64-bit
Byte Order:            Little Endian
CPU(s):                4
On-line CPU(s) list:   0-3
Thread(s) per core:    2
Core(s) per socket:    2
Socket(s):             1
NUMA node(s):          1
Vendor ID:             GenuineIntel
CPU family:            15
Model:                 6
Stepping:              4
CPU MHz:               2992.397
BogoMIPS:              5984.79
Virtualization:        VT-x
L1d cache:             16K
L2 cache:              2048K
NUMA node0 CPU(s):     0-3

**Comment by retep998** [comment]
This seems like the fault of the crypto package for using assembly that apparently older AMD CPUs don't support.

**Comment by AudriusButkevicius** [comment]
This is due to using cloudflare's Go fork with accelerated crypto. Has to be fixed by @calmh

**Comment by LazyGatto** [comment]
Maybe it possible to turn off automatic update for now somehow? To use 11.3 version?

**Comment by LazyGatto** [comment]
For use 11.3 until bug will be resolved - you can disable autoupgrade.
In **config.xml** change `<autoUpgradeIntervalH>` to ***0***
Then replace syncthing binary with 11.3 version.

**Comment by ErichHeer** [comment]
I am sorry but I just replaced the 11.4.exe with 11.3 and restarted syncthing disabled autoupdate and left the office about 1,5 hours ago. 

So I cannot provide you with LOG or similar. I think that I have 6-core Intel CPU. WIN7-64. (But I am not sure)

Interesting: I have upgraded to 11.4 on LENOVO Laptop (INTEL I5 with WIN7-64) and had no problems but with my Office PC I had the crash. 

I could try it at home. Here I am sure to have an AMD Phenom II X4 win WIN7-32.
